tezos_interop/tezos_interop.rei

```diff
@@ -102,6 +102,8 @@ module Context: {
     required_confirmations: int,
   };
 };
+
+type michelson = Micheline.node(int, Michelson_v1_primitives.prim);
```

Comment:
```suggestion
```

Reply:
What do you want here exactly?

Reply:
Attempted addressing this in the incoming commit